gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -751,14 +820,10 @@ def get_network_function_device_status(self, device_data,
             raise exceptions.ComputePolicyNotSupported(
                 compute_policy=device_data['service_details']['device_type'])
 
-        token = self._get_token(device_data.get('token'))
-        if not token:
-            return None
-
         try:
             device = self.compute_handler_nova.get_instance(
-                            token,
-                            self._get_admin_tenant_id(token=token),
+                            device_data['token'],
+                            device_data['tenant_id'],
```

Comment:
Do we get token and tenant_id in device_data ?
According to the blueprint we submitted we don't get key's other than what we have in the check below.
         if (
             any(key not in device_data
                 for key in ['id',
                             'service_details']) or
 
             type(device_data['service_details']) is not dict or
 
             any(key not in device_data['service_details']
                 for key in ['service_vendor',
                             'device_type',
                             'network_mode'])
         ):
             raise exceptions.IncompleteData()

If we want them, then we should update the blueprint.